padinfo/core/transforminfo.py

```diff
@@ -0,0 +1,23 @@
+from padinfo.core.find_monster import findMonsterCustom2
+from padinfo.core.id import get_monster_misc_info
+
+
+async def perform_transforminfo_query(dgcog, raw_query, beta_id3):
```

Comment:
Also you can just put this method on TransformInfoViewState and call it `query` - check out the `EvosViewState` as an example of this

Reply:
I hope I understood correctly. I pulled the misc. info retrieval out of core and put it in `TransformInfoViewState.query`, so now the only thing core does is find the base and transform of a card, should they exist. it feels a little clunky that `TransformInfoViewState.query` only does one thing (call an existing method) though